### ISSUE TITLE
Error with visualization

### ISSUE THREAD

**Issue by Walterbish** [alner/qsSimpleKPI#13]
Hi Alex

By dragging and dropping SimpleKPI object into the edit area I am getting following error message: 

"The visualisation is not found on the server: qsSimpleKPI"

I am using Qlik Sense Version 2.03

Thanks in advance and regards

**Comment by alner** [comment]
Hi, i need more info. Could you please open dev console in your browser and copy errors messages if any?

**Comment by Walterbish** [comment]
Hi Alex

Forgive me but I am not familiar at all on the use of the dev console. Anyway, I have open it, and by selecting the SimpleKPI object in the edit area, it goes to this point of the code, so I guess it may be the error:

<div qv-extension="" class="object-error hard-error ng-scope" tid="object.Error">
	<div class="object-error-content">
		<div class="object-error-title">
			<span class="ng-binding">Visualización no válida</span>
			<i class="object-error-more icon-info" qva-activate="showMore($event)"></i>
		</div>
		<div ng-if="error.message" class="object-error-message ng-scope ng-binding">No se ha encontrado la visualización en el servidor: qsSimpleKPI</div>
		<button ng-if="showButton" class="qui-button object-error-remove ng-scope" qva-activate="remove($event)" q-translation="Common.Delete">Eliminar</button>
	</div>
</div>


Let me know if it is not, and I will try harder


Regards

**Comment by Walterbish** [comment]
<div qv-extension="" class="object-error hard-error ng-scope" tid="object.Error">
	<div class="object-error-content">
		<div class="object-error-title">
			<span class="ng-binding">Visualización no válida</span>
			<i class="object-error-more icon-info" qva-activate="showMore($event)"></i>
		</div>
		<div ng-if="error.message" class="object-error-message ng-scope ng-binding">No se ha encontrado la visualización en el servidor: qsSimpleKPI</div>
		<button ng-if="showButton" class="qui-button object-error-remove ng-scope" qva-activate="remove($event)" q-translation="Common.Delete">Eliminar</button>
	</div>
</div>

**Comment by alner** [comment]
in Chrome you can hit CTRL+SHIFT+J and then select Console tab... then drag and drop qsSimpleKPI...

**Comment by Walterbish** [comment]
I have done as you say, but when dropping qsSimpleKPI object, Console tab remains empty

**Comment by alner** [comment]
Did you import qsSimpleKPI extension using Qlik Management Console?

**Comment by Walterbish** [comment]
Hi Alex

Actually I tried these two links:

build/qsSimpleKPI.zip
https://github.com/alner/qsSimpleKPI.git

The names of the zips I get are diferent (qsSimpleKPI and qsSimpleKPI-master), and I have tried with both of them, using the QMC (separately, I have deleted the first one before importing the second). But the result is the same in both cases: I get the error, and nothing happens in the console tab. 

I must be clearly doing something wrong.

Regards
